tbcrawler/pytbcrawler.py

```diff
@@ -55,6 +57,11 @@ def run():
     job_config = ut.get_dict_subconfig(config, args.config, "job")
     job = crawler_mod.CrawlJob(job_config, url_list)
 
+    # Run display
+    xvfb_wrapper = XvfbWrapper()
+    xvfb_wrapper._from(args.virtual_display)
+    xvfb_wrapper.start()
+
```

Comment:
But you care about the return value of the xvfb_wrapper, no? I don't think there's much difference.
~35 lines vs the ~15 lines, I'd go for 15.

I'd do the input validation in start_xvfb(), but no strong preference.